dxlclient/client_config.py

```diff
@@ -410,6 +447,38 @@ def use_websockets(self, use_websockets):
         self._use_websockets = use_websockets
         self._set_value_to_config(self._USE_WEBSOCKETS_SETTING, use_websockets)
 
+    @property
+    def proxy_address(self):
```

Comment:
Are the various proxy related values something we wish to allow the user to set programmatically and in the DxlClientConfig constructor or will they just be in the config file?

Reply:
Good point. If we want to allow users to set proxy settings programmatically e.g. using env variables 
or by changing global proxy then we need setter methods.

Reply:
I will add setter method if we decide to do that.